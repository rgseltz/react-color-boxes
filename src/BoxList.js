import React from "react";

const BoxList = () => {
    return (
        <div>
            <h1>Box List!</h1>
        </div>
    )
}

export default BoxList;