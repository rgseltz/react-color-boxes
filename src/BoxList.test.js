import React from 'react';
import {render} from '@testing-library/react'
import BoxList from './BoxList';

it('Does not crash', () => {
    render(<BoxList/>)
})

it('matches snapshot', () => {
    const {asFragment} = render(<BoxList/>);
    expect(asFragment()).toMatchSnapshot();
})