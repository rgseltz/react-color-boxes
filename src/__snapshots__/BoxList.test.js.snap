// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Box List!
    </h1>
  </div>
</DocumentFragment>
`;
